p4-fusion/main.cc

```diff
@@ -302,6 +303,9 @@ int Main(int argc, char** argv)
 
 	SUCCESS("Completed conversion of " << totalChanges << " CLs in " << programTimer.GetTimeS() / 60.0f << " minutes, taking " << commitTimer.GetTimeS() / 60.0f << " to commit CLs")
 
+	P4API tagsClient;
```

Comment:
Why do we need to create a new client here? we already have one in the main thread at: https://github.com/sourcegraph/p4-fusion/blob/0603f6ebdee91d52dd36241f407de5e7fdc00ea9/p4-fusion/main.cc#L66

Reply:
In the pipeline when it gets to the setting up the tags the connection in the main thread p4 client seems to have lost the connection, and then it fails when trying to reconnect. Whereas this succeeds 🤷‍♂️